src/blocking/body.rs

```diff
@@ -282,13 +283,13 @@ async fn send_future(sender: Sender) -> Result<(), crate::Error> {
                 buf.reserve(8192);
                 // zero out the reserved memory
                 unsafe {
-                    let uninit = mem::transmute::<&mut [MaybeUninit<u8>], &mut [u8]>(buf.bytes_mut());
+                    let uninit = buf.bytes_mut();
```

Comment:
I would minimize the size of `unsafe` block, `let uninit = buf.bytes_mut();` is not unsafe.